homeassistant/components/media_player/__init__.py

```diff
@@ -277,6 +279,16 @@ def select_source(hass, source, entity_id=None):
     hass.services.call(DOMAIN, SERVICE_SELECT_SOURCE, data)
 
 
+def group_players(hass, source, entity_id=None):
+    """Send the media player the command to group all players."""
+    data = {ATTR_INPUT_SOURCE: source}
```

Comment:
I was not sure data might be containing something else, but I think you're right and it's not needed, entity_id should be the only available option